JitsiConference.js

```diff
@@ -1453,14 +1449,14 @@ JitsiConference.prototype._acceptJvbIncomingCall = function(
 
     // Log "session.restart"
     if (this.wasStopped) {
-        Statistics.sendEventToAll(SESSION_RESTART);
+        Statistics.sendEventToAll(createJingleEvent('restart'));
```

Comment:
I still think it would be useful to keep these event names as constants so that all event names are in one place.